<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>303</width>
     <height>416</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>416</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>1</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="6" column="3">
       <widget class="QPushButton" name="pushButton_Minus">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_0">
        <property name="minimumSize">
         <size>
          <width>122</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>122</width>
          <height>61</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QPushButton" name="pushButton_Plus">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="pushButton_x">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="4">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pushButton_Devide">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
        </property>
        <property name="text">
         <string>รท</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="pushButton_4">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QLabel" name="result">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>244</width>
          <height>61</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
}

background-color : white;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pushButton_9">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_AC">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
        </property>
        <property name="text">
         <string>AC</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButton_7">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QPushButton" name="pushButton_Dot">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_Percent">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
        </property>
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="pushButton_5">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="pushButton_6">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QPushButton" name="pushButton_3">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_PlusMinus">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
        </property>
        <property name="text">
         <string>+/-</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="pushButton_8">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="10" column="3">
       <widget class="QPushButton" name="pushButton_Equal">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="pushButton_1">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>61</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" colspan="4">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0" colspan="4">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="0" colspan="4">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
